sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/AnalyticsTransmissionTarget.java

```diff
@@ -62,6 +62,18 @@
      */
     private final PropertyConfigurator mPropertyConfigurator;
 
+    /**
+     * Transmission interval range values
+     */
+    private static final int TRANSMISSIONINTERVAL_MINMAL = 3;
+    private static final int TRANSMISSIONINTERVAL_MAXMIUM = 24*60;
```

Comment:
24 * 60 * 60

Reply:
Oh, sorry, I will correct this in soon, thanks,